src/components/Checkout/DonationItem.vue

```diff
@@ -307,6 +320,28 @@ export default {
 				this.isCash80Running = true;
 				this.hasCustomDonation = true;
 			}
+
+			// Experiment: CASH-386
+			// eslint-disable-next-line max-len
```

Comment:
not needed here now that breaks are in place.

Reply:
Nice :+1: